ci-operator/step-registry/assisted/baremetal/operator/hypershift/assisted-baremetal-operator-hypershift-workflow.yaml

```diff
@@ -0,0 +1,26 @@
+workflow:
+  as: assisted-baremetal-operator-hypershift
+  steps:
+    allow_skip_on_success: true
+    allow_best_effort_post_steps: true
+    pre:
+    - chain: assisted-baremetal-operator
+    - ref: assisted-baremetal-operator-hypershift
+    post:
+    - ref: assisted-baremetal-operator-gather
+    - ref: baremetalds-packet-gather-metadata
+    - chain: baremetalds-ipi-post
+    env:
+      PACKET_PLAN: "m3.large.x86"
+      USER_DATA_FILENAME: "user-data"
+      DISCONNECTED: "false"
+      NUM_EXTRA_WORKERS: "1"
+      DEVSCRIPTS_CONFIG: |
+        EXTRA_WORKER_VCPU=8
+        EXTRA_WORKER_MEMORY=16384
+        EXTRA_WORKER_DISK=100
+        PROVISIONING_NETWORK_PROFILE=Disabled
+        REDFISH_EMULATOR_IGNORE_BOOT_DEVICE=True
+  documentation: |-
+    Using hub-cluster with Assisted Service operator to deploy extra workers as spoke clusters.
+    Installation will be following HyperShift zero-node flow.
```

Comment:
No docs yet, but I can link to the enhancement for now.

Reply:
ack
btw, that enhancement starts normally and then have all text in bold for some reason :)

Reply:
Any link you can add here as supporting documentation?